samples/highcharts/accessibility/accessible-pattern-fills/demo.html

```diff
@@ -0,0 +1,17 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/series-label.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
```

Comment:
Not using exporting or export data either - can probably remove and simplify options as well.